tests/e2e/taqueria-plugin-flextesa.spec.ts

```diff
@@ -0,0 +1,240 @@
+import {generateTestProject} from "./utils/utils";
+import {execSync} from "child_process";
+import waitForExpect from "wait-for-expect";
+import fs from "fs";
+import {isPortReachable} from "./utils/utils";
+
+
+const taqueriaProjectPath = 'e2e/auto-test-flextesa-plugin';
+let dockerName: string;
+
+describe("E2E Testing for taqueria flextesa plugin",  () => {
+
+    beforeAll(async () => {
+        await generateTestProject(taqueriaProjectPath, ["flextesa"]);
+    })
+
+    test('Verify that taqueria flextesa plugin can start and stop a default sandbox without specifying name', async () => {
+        try {
+
+            // Setting up docker container name
+            dockerName = "local"
+
+            // 1. Run sandbox start command
+            const stdoutStart = execSync(`taq start sandbox`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+
+            // 2. Verify that sandbox has been started and taqueria returns proper message into console
+            expect(stdoutStart).toEqual("Started local.");
+
+            // 3. Verify that docker container has been started
+            const [,dockerContainerTest] = execSync(`docker ps --filter "name=${dockerName}" --no-trunc`).toString().trim().split(/\r?\n/);
```

Comment:
Looks like the comma is a typo here.

Reply:
Hey, @mweichert I have to get the second element in destructuring assignment. That is why it is a correct comma there.

Reply:
We use should an underscore for the variable placeholders which read a little better. This is recommended in the TypeScript Handbook:
https://www.typescriptlang.org/docs/handbook/release-notes/overview.html#destructured-variables-can-be-explicitly-marked-as-unused